honeybee_radiance_command/options/rfluxmtx.py

```diff
@@ -36,3 +39,170 @@ def v(self):
     @v.setter
     def v(self, value):
         self._v.value = value
+
+
+class RfluxmtxControlParameters(object):
+    """Rfluxmtx ControlParameters.
+
+    Set the values for hemisphere sampling type, hemisphere up direction and output file
+    location. This class generates a string for rflumtx which should be included
+    with a receiver aperture group.
+
+    Here is a sample rfluxmtx control parmaters: ``#@rfluxmtx u=0,1,0 h=kf o=output.vmx``
+
+    Args:
+        sampling_type: Set hemisphere sampling type. Acceptable inputs for hemisphere
+            sampling type are:
+             * u for uniform.(Usually applicable for ground).
+             * kf for klems full.
+             * kh for klems half.
+             * kq for klems quarter.
+             * rN for Reinhart - Tregenza type skies. N stands for subdivisions and
+               defaults to 1.
+              * scN for shirley-chiu subdivisions.
```

Comment:
You need line breaks around a bulleted list like this for it to render properly with Sphinx..

Reply:
Also, I think the indent of the last bullet is not correct